### PR TITLE
Remove old and outdated information from the README and make it more presentable.

### LINKED ISSUE #4044: [OTHER] Remove the nonsense (that I believe I wrote) in the README.md as it is quite confusing.
### What would you like to share?

I glanced at the support README and even got lost in contributions I made early in this repo.

As it was my crime to put the nonsense there, I'm going to open a PR that:

- [ ] Converts tabs to spaces
- [ ] Uses Prettier to format
- [ ] Removes HTML <a> links 
- [ ] Removes unnecessary / incorrect text.

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by stemount**
closes #4044

#### What does this PR do?

- Removes confusing information
- Ensures tabs are converted to spaces instead of mismatched space/tabs.
- Markdown Fix/Prettier linting
- Removes some HTML for an <a> link to Markdown format.

#### How can this be manually tested?

Open in GitPod and look at the README.

#### Any background context you want to provide?

I wrote most of the stuff that doesn't make sense.

I moved the socials to the top next to the "active chatters" for better conversion to join discord and subscribe to Eddie's YT channel

#### Screenshots (if appropriate)

Existing README:

##### Existing README

NEWREADME-EDDIEHUB-OLD-PREVIEW

---

##### Proposed README

NEWREADME-EDDIEHUB

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,81 +1,67 @@
...
+- Raise an [issue to join the EddieHub GitHub organisation](https://github.com/EddieHubCommunity/support/issues/new?assignees=&labels=invite+me+to+the+organisation&template=invitation.yml&title=Please+invite+me+to+the+GitHub+Community+Organization).
 
-You volunteer to help that person and pair on it. You fix it and they get super happy.
+- Actions go under [issues](https://github.com/EddieHubCommunity/support/issues)
 
-The unfortunate thing about this is that it lacks a write-up on what was used. It's important to know the articles that were used for reference purposes. This is where a GitHub issue queue comes in. That way, folks could Google for somebody with the same problem!
+  - Bugs
 
-Unfortunately, StackOverflow has a bit of a broken reputation system where folks with the "worst accepted answer" can actually start to get privileges to do more things.
+  - Add me to the organization
```
Comment (schmelto) [inline README.md:23]: Maybe linking the issue here like above.

File: README.md [hunk 2]
```diff
@@ -1,81 +1,67 @@
...
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+
 [![Discord](https://img.shields.io/discord/699608417039286293?style=flat-square)](http://discord.eddiehub.org)
+
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
-Raise an [issue to join the EddieHub GitHub organisation](https://github.com/EddieHubCommunity/support/issues/new?assignees=&labels=invite+me+to+the+organisation&template=invitation.yml&title=Please+invite+me+to+the+GitHub+Community+Organization).
 
-### Some vague idea of how we could turn this into a support channel.
+[Join the Discord Community](http://discord.eddiehub.org) — [Subscribe to Eddie Jaoude's YouTube](https://www.youtube.com/user/eddiejaoude)
+
```
Comment (vijay-jaisankar) [inline README.md:12]: Possible change:
`Subscribe to Eddie Jaoude's YouTube` => `Subscribe to Eddie Jaoude's YouTube channel`

File: README.md [hunk 3]
```diff
@@ -1,81 +1,67 @@
...
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+
 [![Discord](https://img.shields.io/discord/699608417039286293?style=flat-square)](http://discord.eddiehub.org)
+
 <!-- ALL-CONTRIBUTORS-BADGE:END -->
-Raise an [issue to join the EddieHub GitHub organisation](https://github.com/EddieHubCommunity/support/issues/new?assignees=&labels=invite+me+to+the+organisation&template=invitation.yml&title=Please+invite+me+to+the+GitHub+Community+Organization).
 
-### Some vague idea of how we could turn this into a support channel.
+[Join the Discord Community](http://discord.eddiehub.org) — [Subscribe to Eddie Jaoude's YouTube](https://www.youtube.com/user/eddiejaoude)
```
Comment (schmelto) [inline README.md:11]:
```suggestion
[Join the Discord Community](http://discord.eddiehub.org) — [Subscribe to Eddie Jaoude's YouTube channel](https://www.youtube.com/user/eddiejaoude)
```

**Review by schmelto — APPROVED** [review]
## Looking good to me

Your suggested changes looking good to me.
__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant

**Comment by stemount** [comment]
thx @schmelto for committing in the text change. I'll leave the other bits and bobs to be picked up :-P

I mostly wanted to remove my nonsense text

**Pull request merged**